platooning_control/src/platooning_control.cpp

```diff
@@ -45,6 +45,7 @@ namespace platooning_control
     config_.emergency_stop_distance = declare_parameter<double>("emergency_stop_distance", config_.emergency_stop_distance);
     config_.speed_thres_traveling_direction = declare_parameter<double>("speed_thres_traveling_direction", config_.speed_thres_traveling_direction);
     config_.dist_front_rear_wheels = declare_parameter<double>("dist_front_rear_wheels", config_.dist_front_rear_wheels);
+    cmd_timeout_in_s_ = declare_parameter<double>("cmd_timeout_in_s", cmd_timeout_in_s_);
```

Comment:
Might be missing a commit here, but `cmd_timeout_in_s_` will need to be dined in the `ControlPlugin` class for this to build

Reply:
Yup, looks like that didn't get pushed earlier. Just updated it

Reply:
Resolving this comment since we are removing and updates related to the vehicle command timeout from this PR.